alacritty/src/config/bindings.rs

```diff
@@ -412,25 +410,25 @@ macro_rules! bindings {
 
 macro_rules! trigger {
     (KeyBinding, $key:literal, $location:expr) => {{
-        BindingKey::Keycode { key: Character($key.into()), location: $location }
+        BindingKey::Keycode { key: Key::Character($key.into()), location: $location }
     }};
     (KeyBinding, $key:literal,) => {{
-        BindingKey::Keycode { key: Character($key.into()), location: KeyLocation::Standard }
+        BindingKey::Keycode { key: Key::Character($key.into()), location: KeyLocation::Standard }
     }};
-    (KeyBinding, $key:expr,) => {{
-        BindingKey::Keycode { key: Named($key), location: KeyLocation::Standard }
+    (KeyBinding, $key:ident,) => {{
+        BindingKey::Keycode { key: Key::Named(NamedKey::$key), location: KeyLocation::Standard }
     }};
-    ($ty:ident, $key:expr,) => {{
-        $key
+    (MouseBinding, $button:ident,) => {{
+        MouseButton::$button
     }};
 }
 
 pub fn default_mouse_bindings() -> Vec<MouseBinding> {
     bindings!(
         MouseBinding;
-        MouseButton::Right;                            MouseAction::ExpandSelection;
```

Comment:
Honestly preferred the old one, how is anyone supposed to figure this out with just `Right`?

Reply:
It worked just fine before, didn't it?

Reply:
Because of `expr`, but `Right` is not expr, since it complains that the type is not here, so you need either `ident` or `tt`, but you can't use `ident`, because you have literals, so the only option is `tt`, but `::` is token, so you get the issue...

Reply:
The macro is common with keyboard which has literals.